### ISSUE TITLE
new sdk changes Directus 9.5.2 

### ISSUE THREAD

**Issue by fballan93** [apstanisic/directus-dart#35]
hey the new sdk switched the SDKs type signature was tweaked to make it consistent with the API's internal services. The .readMany method has been renamed to .readByQuery. 

will this SDK receive an update to reflect this?

**Comment by apstanisic** [comment]
I'll check it out. I think they changed their license from MIT to GPLv3, so I'll see what can be done. 
Is REST API is the same?

**Comment by fballan93** [comment]
yes REST has not changed . however their sdk  have a potential breaking change which is The .readMany method has been renamed to .readByQuery.

another thing which is in version 0.7.0 the log in is still not working
